### PR TITLE
feat(platform): hydrate WorkItem.body; route two-pass scan through adapter

### PR DESCRIPTION
**Pull request by babonet**
Phase 2 of routing the watch CLI through PlatformAdapter so ADO-first squads work.

- WorkItem gains optional body; populated by GitHubAdapter (gh issue view --json body) + AzureDevOpsAdapter (System.Description)
- watch two-pass hydrates via adapter.getWorkItem instead of hardcoded gh issue view; drops the gh requirement
- changeset (sdk minor, cli patch)

Validation: SKIP_BUILD_BUMP=1 npm run build; vitest (184 pass). Independent of #1413.

### CODE REVIEW

File: packages/squad-cli/src/cli/commands/watch/capabilities/two-pass.ts [hunk 1]
```diff
@@ -45,15 +41,12 @@ export class TwoPassCapability implements WatchCapability {
...
-          const { stdout: detailJson } = await execFileAsync('gh', [
-            'issue', 'view', String(item.id),
-            '--json', 'number,title,body,labels,assignees',
-          ], { maxBuffer: 5 * 1024 * 1024 });
-          hydrated.push(JSON.parse(detailJson));
+          const full = await context.adapter.getWorkItem(item.id);
+          hydrated.push({ number: full.id, title: full.title, body: full.body });
         } catch {
           hydrated.push({ number: item.id, title: item.title });
         }
```
[the remark this answers is not in the document]
Comment (babonet) [reply packages/squad-cli/src/cli/commands/watch/capabilities/two-pass.ts]: Fixed in 06dd1476 — two-pass now reuses item.body when present and only calls getWorkItem when missing, so ADO no longer re-fetches each actionable item.

**Review by tamirdresher — APPROVED** [review]
✅ Approved — surgical and correct. Removes hardcoded gh CLI dependency from two-pass, routes through adapter. body field added with proper type guards (ADO typeof check is defensive coding done right). requires:[] is correct since the adapter owns the dependency now. Narrowed hydrated shape matches the declared type better than the old raw JSON push. No security concerns. Clean.

**Review by tamirdresher — COMMENTED** [review]
Reviewed the platform-adapter change — LGTM overall, all CI green.

- `WorkItem.body?` is purely additive; no existing consumer breaks.
- GitHub path: `listWorkItems` doesn't return `body`, so items fall through to `getWorkItem` — 1 API call per actionable item, identical to the removed hardcoded `gh issue view`. No regression.
- ADO path: `listWorkItems` already hydrates `body` via per-item `getWorkItem`, so the `item.body !== undefined` short-circuit correctly avoids a duplicate fetch.
- `requires: ['gh'] -> []` is not a preflight regression — `requires` is display metadata here; the preflight loop only calls `cap.preflight()`.

Non-blocking suggestions:
1. Consider bumping the CLI changeset `patch -> minor` — enabling ADO for this capability is a user-visible feature.
2. Add a `TwoPassCapability` unit test (assert `requires` empty, `phase: 'post-triage'`) to match sibling capabilities.
3. Style: `readonly requires: readonly string[] = []` for explicitness.

**Comment by tamirdresher** [comment]
@babonet — this one is in good shape: MERGEABLE, all CI green. When you get a chance, could you rebase onto latest `dev`, squash the two commits into one, and apply the three non-blocking follow-ups from my earlier review before we move to merge?

1. Bump the `@github/copilot` changeset from `patch` → `minor` — enabling ADO for the two-pass capability is a user-visible feature, not a bugfix.
2. Add a focused `TwoPassCapability` unit test asserting `requires` is empty and `phase === 'post-triage'`, matching the sibling capabilities' test shape.
3. Style: declare the field as `readonly requires: readonly string[] = []` for explicitness (both the property and the array element type).

Once those land I'll take another look. Thanks!

**Comment by bradygaster** [comment]
This remains in the approved owner-update queue. Before merge, please rebase onto current dev, squash the two commits, apply the three follow-ups already listed by @tamirdresher (CLI changeset minor bump, focused TwoPassCapability test, explicit readonly type), and let the full modern CI suite run. We will re-review promptly once those changes land.

**Review by tamirdresher — APPROVED** [review]
Reviewed the adapter-routing change in detail and spot-checked the affected call paths.

- `WorkItem.body?` is additive and backward-compatible for existing consumers.
- `two-pass` now hydrates through `context.adapter.getWorkItem()` instead of shelling out to `gh issue view`, which correctly removes the GitHub-only dependency from this capability.
- The `item.body !== undefined` short-circuit avoids an unnecessary second fetch for adapters like Azure DevOps that already populate body during listing.
- GitHub adapter change is minimal (`body` added to the existing `gh issue view --json ...` contract), and the ADO adapter guards `System.Description` with a string check before exposing it.
- I did not find a regression in the error path: failed hydration still degrades to `{ number, title }`, matching the prior behavior.

Given the narrow scope plus green CI, this looks safe to merge.
